stdlib/asyncio/locks.pyi

```diff
@@ -19,13 +18,12 @@ if sys.version_info >= (3, 9):
 else:
     class _ContextManager:
         def __init__(self, lock: Lock | Semaphore) -> None: ...
-        def __enter__(self) -> object: ...
+        def __enter__(self) -> None: ...
         def __exit__(self, *args: Any) -> None: ...
 
     class _ContextManagerMixin:
-        def __init__(self, lock: Lock | Semaphore) -> None: ...
```

Comment:
`_ContextManagerMixin` doesn't have a constructor - it looks like it was copied from `_ContextManager`